### PR TITLE
chore: update readme with new instructions for UI

### PR DESCRIPTION
**Pull request by HarikrishnanBalagopal**
Signed-off-by: Harikrishnan Balagopal <harikrishmenon@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,9 +41,12 @@ brew install move2kube
 
 To bring up UI version:
 
-* Change directory to an empty directory using say, `mkdir -p workspace && cd workspace`
-* Run `docker run -p 8080:8080 -v $PWD:/workspace -v /var/run/docker.sock:/var/run/docker.sock -it quay.io/konveyor/move2kube-aio:latest`
-* Access the UI in `http://localhost:8080/`.
+```
+docker run --rm -it -p 8080:8080 -v "${PWD}/data:/move2kube-api/data" -v /var/run/docker.sock:/var/run/docker.sock quay.io/konveyor/move2kube-ui:latest
```
Comment (ashokponkumar) [inline README.md:45]: Let's remove the docker.sock. its only for advanced usecases.

File: README.md [hunk 2]
```diff
@@ -41,9 +41,12 @@ brew install move2kube
...
+```
+Then go to http://localhost:8080 in a browser
+
+More detailed instructions, including how to build the image from source, can be found here: https://github.com/konveyor/move2kube-ui#starting-the-ui
```
Comment (ashokponkumar) [inline README.md:49]: Make 'here' a link, instead of full url.

**Review by ashokponkumar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (HarikrishnanBalagopal) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (HarikrishnanBalagopal) [reply README.md:49]: It's good practise not to hide links. People like to see where they are being taken to.

**Review by ashokponkumar — APPROVED** [review]

**Pull request merged**